boards/README.md

```diff
@@ -0,0 +1,43 @@
+# Board Support Modules
+
+
+## generic_st7735
+
+Module | Description
+------ | -----------
+landscape_128x128.py | st7735 128x128 display in landscape orientation |
```

Comment:
Continuing [my previous comment](https://github.com/lvgl/lv_binding_micropython/pull/198#issuecomment-988257500),  does it make sense to create different modules for landscape/portrait? Why not provide a single module the the user could import and then configure? There could be other parameters the user would want to change.